### PR TITLE
fix: Id of an entity is used for entityId instead of processInstanceId 

### PR DESCRIPTION
**Pull request by CTI777**

### CODE REVIEW

**Comment by CTI777** [comment]
- [ ] CloudSequenceFlowTakenImpl
_Not sure if here it is correct_ 
**Related tests:** 
/activiti-cloud-runtime-bundle-service/activiti-cloud-services-runtime-bundle/activiti-cloud-services-connectors/src/test/java/org/activiti/services/connectors/channel/ServiceTaskIntegrationResultEventHandlerTest.java

- [ ] CloudIntegrationResultReceivedImpl
**Related tests:** 
/activiti-cloud-runtime-bundle-service/activiti-cloud-services-runtime-bundle/activiti-cloud-services-connectors/src/test/java/org/activiti/services/connectors/channel/ServiceTaskIntegrationResultEventHandlerTest.java

- [ ] CloudBPMNActivityEventImpl
**Dependent:**
CloudBPMNActivityCancelledEventImpl
CloudBPMNActivityCompletedEventImpl
CloudBPMNActivityStartedEventImpl
**Related tests:** 
/activiti-cloud-audit-service/activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/AuditServiceIT.java

- [ ] CloudIntegrationRequestedImpl
**Related tests:** 
/activiti-cloud-runtime-bundle-service/activiti-cloud-services-runtime-bundle/activiti-cloud-services-connectors/src/test/java/org/activiti/services/connectors/IntegrationRequestSenderTest.java

File: activiti-cloud-api-process-model-impl/src/main/java/org/activiti/cloud/api/process/model/impl/events/CloudBPMNActivityEventImpl.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ public CloudBPMNActivityEventImpl(BPMNActivity entity,
         super(entity);
         this.processDefinitionId = processDefinitionId;
         this.processInstanceId = processInstanceId;
-        setEntityId(processInstanceId);
+        setEntityId(entity.getElementId());
```
Comment (erdemedeiros) [inline activiti-cloud-api-process-model-impl/src/main/java/org/activiti/cloud/api/process/model/impl/events/CloudBPMNActivityEventImpl.java:37]: @CTI777 the right value for this is `entity.getId()`. But this field is missing in the API. I created a new issue to add it: https://github.com/Activiti/Activiti/issues/2175.

File: activiti-cloud-api-process-model-impl/src/main/java/org/activiti/cloud/api/process/model/impl/events/CloudBPMNActivityEventImpl.java [hunk 2]
```diff
@@ -47,7 +47,7 @@ public CloudBPMNActivityEventImpl(String id,
               entity);
         this.processDefinitionId = processDefinitionId;
         this.processInstanceId = processInstanceId;
-        setEntityId(processInstanceId);
+        setEntityId(entity.getElementId());
```
Comment (erdemedeiros) [inline activiti-cloud-api-process-model-impl/src/main/java/org/activiti/cloud/api/process/model/impl/events/CloudBPMNActivityEventImpl.java:50]: Same as before.

File: activiti-cloud-api-process-model-impl/src/main/java/org/activiti/cloud/api/process/model/impl/events/CloudSequenceFlowTakenImpl.java [hunk 3]
```diff
@@ -28,7 +28,8 @@ public CloudSequenceFlowTakenImpl() {
 
     public CloudSequenceFlowTakenImpl(SequenceFlow sequenceFlow) {
         super(sequenceFlow);
-        setEntityId(sequenceFlow.getProcessInstanceId());
+        if (getEntity()!=null)
+            setEntityId(getEntity().getSourceActivityElementId());
```
Comment (erdemedeiros) [inline activiti-cloud-api-process-model-impl/src/main/java/org/activiti/cloud/api/process/model/impl/events/CloudSequenceFlowTakenImpl.java]: Here, you also need to use `getEntity.getId()`after fixing https://github.com/Activiti/Activiti/issues/2175.

Comment (CTI777) [reply activiti-cloud-api-process-model-impl/src/main/java/org/activiti/cloud/api/process/model/impl/events/CloudSequenceFlowTakenImpl.java]: 1. Do we need this for BPMNElement (as it is stated in Issue) or BPMNActivity?
public interface BPMNActivity extends BPMNElement
2. setEntityId(getEntity().getSourceActivityElementId());
For SequenceFlow we have SourceActivity, and Target...
O rI really have to write: setEntityId(getEntity().getEntity.getId());

Comment (salaboy) [reply activiti-cloud-api-process-model-impl/src/main/java/org/activiti/cloud/api/process/model/impl/events/CloudSequenceFlowTakenImpl.java]: @CTI777 this is related to the closed PR in Activiti-api, here you should use SequenceFlow.getElementId() in the same way as the BPMNActivity after changing that in Activiti API

**Pull request merged**